packages/synapse-react-client/src/components/QueryWrapperPlotNav/QueryWrapperSynapsePlot.tsx

```diff
@@ -61,12 +67,23 @@ export default function QueryWrapperSynapsePlot(
       {showFacetVisualization && (
         <>
           {title && (
-            <FacetFilterHeader
-              hideCollapsible={true}
-              label={title}
-              isCollapsed={false}
-              onClick={() => {}}
-            />
+            <div className="FacetNavPanel__title">
+              <span className="FacetNavPanel__title__name">{title}</span>
+              <div role="toolbar" className="FacetNavPanel__title__tools">
+                <Tooltip title={'Hide graph under Show More'}>
+                  <IconButton
+                    onClick={() => {
+                      if (onHide) {
+                        onHide()
+                      }
+                    }}
+                    size={'small'}
+                  >
+                    <IconSvg icon={'close'} wrap={false} fontSize={'inherit'} />
+                  </IconButton>
+                </Tooltip>
+              </div>
+            </div>
```

Comment:
Let's try to extract out a `PlotPanelHeader` component and use it here + in FacetNavPanel